packages/ui/src/common/components/AboutDateColumn.tsx

```diff
@@ -12,24 +14,41 @@ import { TextMedium, TextSmall } from './typography'
 interface Props {
   block: Block
   time: string
+  horizontal?: boolean
 }
 
-export const AboutDateColumn = React.memo(({ time, block }: Props) => (
-  <Column>
+export const AboutDateColumn = React.memo(({ time, block, horizontal }: Props) => (
+  <Wrapper horizontal={horizontal}>
     <AboutText>{formatDateString(time)}</AboutText>
+    {horizontal && <Separator>|</Separator>}
```

Comment:
After the changes in where we get the timestamp, I think that we should also use only `Block` here and use its timestamp.